galaxy_ng/app/api/ui/viewsets/execution_environment.py

```diff
@@ -215,11 +215,15 @@ def get_object(self):
 class ContainerRegistryRemoteFilter(filterset.FilterSet):
     name = filters.CharFilter(field_name='name')
     url = filters.CharFilter(field_name='url')
+    created_at = filters.CharFilter(field_name='created_at')
```

Comment:
I think the field name should be `pulp_created` and `pulp_last_updated`.

Have you tested this?

Reply:
Do you mean tested in the UI?